mailpoet/assets/js/src/automation/listing/cells/Status.tsx

```diff
@@ -0,0 +1,22 @@
+import { __ } from '@wordpress/i18n';
+import { Toggle } from '../../../common';
+import { Workflow, WorkflowStatus } from '../workflow';
+
+type Props = {
+  workflow: Workflow;
+};
+
+export function Status({ workflow }: Props): JSX.Element {
+  const toggleStatus = () => {
+    // @Todo
+  };
+
+  return (
+    <div>
+      <Toggle onCheck={toggleStatus} />
```

Comment:
Seems like you're using a toggle component from the MailPoet design system. We need to use the toggle from `@wordpress/components`.